lib/cog_api/resources/trigger.ex

```diff
@@ -1,13 +1,14 @@
 defmodule CogApi.Resources.Trigger do
   @derive [Poison.Encoder]
 
-  defstruct [:id,
-             :name,
-             :pipeline,
-             :as_user,
-             :timeout_sec,
-             :description,
-             :invocation_url]
+  defstruct [id: nil,
```

Comment:
@drapergeek I added the `enabled` field, which defaults to true, and I can't mix bare atoms and Keywords.